rcli/display.py

```diff
@@ -32,7 +32,7 @@
 from tqdm import tqdm
 
 try:
-    from shutils import get_terminal_size
+    from shutil import get_terminal_size
```

Comment:
I discovered this bug while attempting to figure out why Travis was blowing up on 2.7 and 3.3.